Epoch 1: Train Loss: nan, Train Vis Loss: nan, Val Loss: nan, Train Acc: 60.84%, Val Accuracy: 59.63%, Train F1 Score: 15.14%, Val F1 Score: 14.94%
For class normal the Train F1: 75.65% and Val F1: 74.71%
For class AVNRT the Train F1: 0.06% and Val F1: 0.00%
For class AVRT the Train F1: 0.00% and Val F1: 0.00%
For class concealed the Train F1: 0.00% and Val F1: 0.00%
For class EAT the Train F1: 0.00% and Val F1: 0.00%
Time taken for epoch 1: 227.29 minutes
Epoch 2: Train Loss: nan, Train Vis Loss: nan, Val Loss: nan, Train Acc: 60.88%, Val Accuracy: 59.63%, Train F1 Score: 15.14%, Val F1 Score: 14.94%
For class normal the Train F1: 75.68% and Val F1: 74.71%
For class AVNRT the Train F1: 0.00% and Val F1: 0.00%
For class AVRT the Train F1: 0.00% and Val F1: 0.00%
For class concealed the Train F1: 0.00% and Val F1: 0.00%
For class EAT the Train F1: 0.00% and Val F1: 0.00%
Time taken for epoch 2: 226.92 minutes
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x7054b7a52200>
Traceback (most recent call last):
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/site-packages/torch/utils/data/dataloader.py", line 1479, in __del__
    self._shutdown_workers()
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/site-packages/torch/utils/data/dataloader.py", line 1443, in _shutdown_workers
    w.join(timeout=_utils.MP_STATUS_CHECK_INTERVAL)
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/multiprocessing/process.py", line 149, in join
    res = self._popen.wait(timeout)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/multiprocessing/popen_fork.py", line 40, in wait
    if not wait([self.sentinel], timeout):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/multiprocessing/connection.py", line 1135, in wait
    ready = selector.select(timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/selectors.py", line 415, in select
    fd_event_list = self._selector.poll(timeout)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt:
Traceback (most recent call last):
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/runpy.py", line 198, in _run_module_as_main
    return _run_code(code, main_globals, None,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/runpy.py", line 88, in _run_code
    exec(code, run_globals)
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/__main__.py", line 39, in <module>
    cli.main()
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 430, in main
    run()
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 284, in run_file
    runpy.run_path(target, run_name="__main__")
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 321, in run_path
    return _run_module_code(code, init_globals, run_name,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 135, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 124, in _run_code
    exec(code, run_globals)
  File "/home/tzikos/Desktop/KULeuvenThesis/main.py", line 110, in <module>
    CVtrain(modelStr=MODEL_STR, learningRate=LEARNING_RATE_BERT, epochs=1000, classWeights=classWeights,
  File "/home/tzikos/Desktop/KULeuvenThesis/models.py", line 717, in CVtrain
    loss.backward()
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/site-packages/torch/_tensor.py", line 522, in backward
    torch.autograd.backward(
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/site-packages/torch/autograd/__init__.py", line 266, in backward
    Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
KeyboardInterrupt